src/Mod/BIM/bimcommands/BimMaterial.py

```diff
@@ -522,6 +520,11 @@ def rescan(self, rebuild=False):
                     i.setFlags(i.flags() | QtCore.Qt.ItemIsEditable)
                     if o.Name == name:
                         self.dlg.matList.setCurrentItem(i)
+            if hasattr(self.dlg, "buttonMergeDupes"):
+                if len(self.dlg.materials) > 1:
+                    self.dlg.buttonMergeDupes.setEnabled(True)
+                else:
+                    self.dlg.buttonMergeDupes.setEnabled(False)
```

Comment:
the if statement is not needed here:
```suggestion
                hasMultipleMaterials = len(self.dlg.materials) > 1
                self.dlg.buttonMergeDupes.setEnabled(hasMultipleMaterials)
```